index.js

```diff
@@ -33,8 +33,55 @@ client.on('message', msg => {
             port: config.apiPort,
             path: 'memes/hot'
         }, sendMeme)
+    } else if (message === 'send wholesome') {
+        http.get({
+            hostname: config.apiHost,
+            port: config.apiPort,
+            path: 'wholesomememes/new'
+        }, sendMeme)
+    } else if (message === 'send hot wholesome') {
+        http.get({
+            hostname: config.apiHost,
+            port: config.apiPort,
+            path: 'wholesomememes/hot'
+        }, sendMeme)
+    } else if (message === 'send dank') {
+        http.get({
+            hostname: config.apiHost,
+            port: config.apiPort,
+            path: 'dankmemes/new'
+        }, sendMeme)
+    } else if (message === 'send hot dank') {
+        http.get({
+            hostname: config.apiHost,
+            port: config.apiPort,
+            path: 'dankmemes/hot'
+        }, sendMeme)
+    } else if (message === 'send anime') {
```

Comment:
The bot is not sending actual anime isn't it? It'd be more intuitive for the command to be "send anime meme" or something.